app/json/slate_lineup_configs.json

```diff
@@ -286,7 +288,9 @@
     "experiments": [
       {
         "description": "Explore Topic SlateLineup",
-        "rankers": [],
+        "rankers": [
+          "thompson-sampling"
+        ],
```

Comment:
TODO: Revert these changes before deployment. They're for debugging in Pocket-Dev only.